orm_lib/tests/db_test.cc

```diff
@@ -119,12 +120,13 @@ void doPostgreTest(const drogon::orm::DbClientPtr &clientPtr)
                        "postgresql - DbClient streaming-type interface(0)");
         };
     /// 1.2 insert,blocking
-    *clientPtr << "insert into users (user_id,user_name,password,org_name) "
-                  "values($1,$2,$3,$4) returning *"
-               << "pg1"
-               << "postgresql1"
-               << "123"
-               << "default" << Mode::Blocking >>
+    *clientPtr
```

Comment:
Please make a test case in that the default value is in the middle of the parameters list. you could simply move the admin column ahead.